src/hooks/useDataset/index.js

```diff
@@ -26,6 +26,14 @@ import { getSolidDataset } from "@inrupt/solid-client";
 export const GET_DATASET = "getDataset";
 
 export default function useDataset(iri) {
+  const fetchDataset = (datasetIri, fetch) => {
+    if (!datasetIri) {
+      return null;
+    }
+    return getSolidDataset(datasetIri, { fetch });
+  };
   const { fetch } = useSession();
-  return useSWR([iri, GET_DATASET], () => getSolidDataset(iri, { fetch }));
+  return useSWR([iri, GET_DATASET], () => fetchDataset(iri, fetch), {
+    refreshInterval: 0,
```

Comment:
I think these should be options we pass to `useDataset`, and which is then passed on to `useSWR`. That way we get more flexibility for each use of `useDataset`.

Reply:
Make useDataset more flexible by allowing the SWR options to be passed. We don't necessarily want _all_ uses of `useDataset` to have `refreshInterval: 0` after all